### PR TITLE
Update tooltips.md

### PR DESCRIPTION
**Pull request by zeinjaber**
Related to SPM-1647. Suggesting more explanation for "Visitor Solve Rate" and "API Solve Rate".

### CODE REVIEW

File: content/turnstile/reference/tooltips.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ layout: single
...
+**Visitor Solve Rate**: Considers all humans who have viewed the widget, whether it was interactive or non-interactive. It measures the percentage of people who completed the widget. For example, if 100 people saw the widget and 20 of them completed it successfully, the Visitor Solve Rate would be 20%. [Learn more](https://developers.cloudflare.com/turnstile/frequently-asked-questions/#what-is-visitor-solve-rate).
 
-**API Solve Rate**: The percentage of humans that the website sees who have successfully completed the widget.
+**API Solve Rate**: Focuses only on humans who interacted with the widget through the website's API. It measures the percentage of successful widget completions among all visitors detected through the API. This metric looks specifically at the number of people who successfully completed the widget. For example, if the website detected 200 visitors through its API and 50 of them completed the widget, the API Solve Rate would be 25%. [Learn more](https://developers.cloudflare.com/turnstile/frequently-asked-questions/#what-is-visitor-solve-rate).
```
Comment (punkeel) [inline content/turnstile/reference/tooltips.md]: Hm, not sure - API Solve Rate is what we observe on /siteverify.
--> For example, if 100 people saw the widget, 20 of them completed it successfully, and only 15 of them used their token (eg: submitted the form), the API Solve Rate would be 15%

Comment (zeinjaber) [reply content/turnstile/reference/tooltips.md]: Thanks, @punkeel, for reviewing my PR. I added your example to the PR. Please let me know if there's anything else needs to be corrected. Thanks.

File: content/turnstile/reference/tooltips.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ layout: single
...
+**Visitor Solve Rate**: Considers all humans who have viewed the widget, whether it was interactive or non-interactive. It measures the percentage of people who completed the widget. For example, if 100 people saw the widget and 20 of them completed it successfully, the Visitor Solve Rate would be 20%. [Learn more](https://developers.cloudflare.com/turnstile/frequently-asked-questions/#what-is-visitor-solve-rate).
 
-**API Solve Rate**: The percentage of humans that the website sees who have successfully completed the widget.
+**API Solve Rate**: Focuses only on humans who interacted with the widget through the website's API. It measures the percentage of successful widget completions among all visitors detected through the API. This metric looks specifically at the number of people who successfully completed the widget. For example, if 100 people viewed the widget, 20 of them completed it successfully, and only 15 of them used their token (eg: submitted the form), the API Solve Rate would be 15%. [Learn more](https://developers.cloudflare.com/turnstile/frequently-asked-questions/#what-is-visitor-solve-rate).
```
Comment (patriciasantaana) [inline content/turnstile/reference/tooltips.md]:
```suggestion
**API Solve Rate**: Focuses only on humans who interacted with the widget through the website's API. It measures the percentage of successful widget completions among all visitors detected through the API. This metric looks specifically at the number of people who successfully completed the widget. For example, if 100 people viewed the widget, 20 of them completed it successfully, and only 15 of them used their token, the API Solve Rate would be 15%. [Learn more](/turnstile/frequently-asked-questions/#what-is-visitor-solve-rate).
```

File: content/turnstile/reference/tooltips.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ layout: single
 ---
 # Tooltips
 
-**Visitor Solve Rate**: The percentage of humans who have seen the widget and completed it. This can be both interactive and non-interactive.
+**Visitor Solve Rate**: Considers all humans who have viewed the widget, whether it was interactive or non-interactive. It measures the percentage of people who completed the widget. For example, if 100 people saw the widget and 20 of them completed it successfully, the Visitor Solve Rate would be 20%. [Learn more](https://developers.cloudflare.com/turnstile/frequently-asked-questions/#what-is-visitor-solve-rate).
```
Comment (patriciasantaana) [inline content/turnstile/reference/tooltips.md]:
```suggestion
**Visitor Solve Rate**: Considers all humans who have viewed the widget, whether it was interactive or non-interactive. It measures the percentage of people who completed the widget. For example, if 100 people saw the widget and 20 of them completed it successfully, the Visitor Solve Rate would be 20%. [Learn more](/turnstile/frequently-asked-questions/#what-is-visitor-solve-rate).
```

**Review by patriciasantaana — APPROVED** [review]

**Pull request merged**